packages/amplify_datastore/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   amplify_core: 0.1.0
   plugin_platform_interface: ^1.0.1
   date_time_format: ^1.1.0
-  uuid: ^2.2.2
```

Comment:
Is the `uuid` dependency needed in the `amplify_datastore` package when its already a dependency in the `amplify_datastore_plugin_interface` package?

Reply:
Thanks for the comment.  It’s a good point, the datastore package doesn’t need uuid.  I just tested and validated removing it won’t cause any problems.  I’ll remove the package.